docs/user-guide/notifications/notification-settings/README.md

```diff
@@ -5,26 +5,66 @@ sidebar_navigation:
+You can configure how and for what events you wish to be notified through Notification Center. To access these settings, you can either click on **_your avatar on the top right corner → My account → Notification settings_** or click on **Notification Settings** on the top right corner of Notification center.
 
-![notification-settings](notification-settings.png)
+>> IMG
 
-In the **Notification settings** you can fine-tune what you are notified about:
+## Participating
 
-- **I am @mentioned**: You can switch notification on/off  when somebody tags you in a comment (i.e. @YOU)
-- **Assigned to me or accountable:** You can switch notification on/off when you are either assigned or made accountable for a task, for example.
-- **Also notify me for:** Here you can further finetune under what circumstances you would like to be notified, or not:
-  - Updates on watched items:
-  - New work packages
-  - All status changes
-  - All date changes
-  - All priority changes
-  - All new comments
+You participate in a work package by either being [mentioned](../../work-packages/edit-work-package/#-notification-mention), by watching it (being on the _Watchers_ list) or by being designated assignee or accountable. 
 
-When someone [mentions](../../work-packages/edit-work-package/#-notification-mention) you (@xxx) in e.g. a work package description or comment you will receive an in-app notification.
+By default, you will be notified of all activities in work packages in which you participate. However, you can choose to disable these notifications for work packages for which you are assignee or accountable by unchecking these options:
 
-Additionally, you can also add **project specific notification settings** by clicking on the link **Add settings for project**.
+>> IMG
 
-By default you do not receive any notifications about your own changes. In addition to the in-app notifications, you will also get a once-a-day summary of all notifications as **[Email reminders](../../../getting-started/my-account#email-reminders)**.
+You cannot disable notifications for when you are mentioned (since the goal of mentioning you is to get your attention) or for work packages that you are watching. For the latter, you may disable further notifications simply by unwatching those work package.
+
+> Info: Modifying these settings might result in your missing updates and changes that are relevant to you. We do not recommend changing them unless you are absolutely certain of the consequences.
+
+## Date alerts
+
+Starting 12.4, Open Project offers notification for date alerts. Please note that this is an Enterprise feature.
+
+> **Note**: Date alerts are a Premium Feature and can only be used with [Enterprise cloud](../../enterprise-guide/enterprise-cloud-guide/) or [Enterprise on-premises](../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free Community Edition is easy and helps support OpenProject.
```

Comment:
The links to the enterprise guides need to go one more level up:
`> **Note**: Date alerts are a Premium Feature and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free Community Edition is easy and helps support OpenProject.
`

Reply:
Thank you @as-op, fixed!